source/Nuke.Common/Tools/SignTool/SignToolSettings.cs

```diff
@@ -20,18 +19,12 @@ private static string GetToolPath()
                 : SpecialFolders.ProgramFiles).NotNull();
 
         var platformIdentifier = EnvironmentInfo.Is64Bit ? "x64" : "x86";
+        const string windowsKitLastVersion = "10";
 
-        return new[]
-               {
-                   programDirectory / "Windows Kits" / "10" / "bin" / "10.0.15063.0",
-                   programDirectory / "Windows Kits" / "10" / "App Certification Kit",
-                   programDirectory / "Windows Kits" / "10" / "bin" / platformIdentifier,
-                   programDirectory / "Windows Kits" / "8.1" / "bin" / platformIdentifier,
-                   programDirectory / "Windows Kits" / "8.0" / "bin" / platformIdentifier,
-                   programDirectory / "Microsoft SDKs" / "Windows" / "v7.1A" / "Bin"
-               }
-            .Select(x => x / "signtool.exe")
-            .WhereFileExists()
-            .FirstOrDefault();
+        var windowsKitsRootDirectory = programDirectory / "Windows Kits" / windowsKitLastVersion;
+
+        var signToolPath = windowsKitsRootDirectory.GlobFiles($"bin/{windowsKitLastVersion}.*/{platformIdentifier}/signtool.exe").LastOrDefault();
```

Comment:
Shouldn't we order by version number first before using `LastOrDefault`?

Reply:
the order returned is naturally the alphabetical order, meaning version 10.0.22 will be after version 10.0.17.

Reply:
Thanks for the info, I did not know it's already ordered. 👍